3-Authorization-II/groups/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,84 @@
+{
+    "Sample": {
+        "Title": "Enable your Java Spring boot web app to restrict access to routes using security groups with the Microsoft identity platform",
+        "Level": 100,
+        "Client": "Java Spring Boot web app",
+        "RepositoryUrl": "ms-identity-java-spring-tutorial",
+        "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+        {
+            "Id": "webApp",
+            "Name": "java-spring-webapp-groups",
+            "Kind": "WebApp",
+            "Audience": "AzureADMyOrg",
+            "PasswordCredentials": "Auto",
+            "AllowImplicitFlow": false,
+            "HomePage": "http://localhost:8080/",
+            "ReplyUrls": "http://localhost:8080/login/oauth2/code/",
+            "GroupMembershipClaims": "SecurityGroup",
+            "RequiredResourcesAccess": [
+                {
+                    "Resource": "Microsoft Graph",
+                    "DelegatedPermissions": [
+                        "GroupMember.Read.All"
+                    ]
+                }
+            ],
+            "SecurityGroups": [
+                {
+                    "Name": "AdminGroup",
+                    "Description": "Admin Security Group"
+                },
+                {
+                    "Name": "UserGroup",
+                    "Description": "User Security Group"
+                }
+            ]
```

Comment:
>] [](http://example.com/codeflow?start=12&length=1)

add manual step. :"Navigate to the portal and provde admin consent to grp..read.all"